### ISSUE TITLE
How my network can be convergent quickly ?

### ISSUE THREAD

**Issue by yaoanderson** [sowson/darknet#14]
Hi @sowson ,

I have trained my network based on issue "Please kindly help us about not convergent network", after 412 rounds the "obj" still keep on one low value (0.00xxx), so how can I make it convergent quickly ? And I see you have trained this network before, how many round did your network can be convergent to < 5.0 avg ? Thanks
![image](https://user-images.githubusercontent.com/4134275/58366571-dca76500-7f06-11e9-8b54-12046ddacc62.png)

**Comment by yaoanderson** [comment]
I just want to confirm I am on the right way, maybe I need to train it > 1000 rounds first, then to check my issue is existing or not, close this issue now.

**Comment by yaoanderson** [comment]
After 750 rounds training I found obj value still < 0.00xxx, is there any thing wrong ? Could you help us about this slow convergent issue ? @sowson 
![image](https://user-images.githubusercontent.com/4134275/58376017-22aa0a80-7f93-11e9-8dc0-95ed4f417117.png)

**Comment by sowson** [comment]
I need to think about it more... the thing is that trained CNN works even if loss not showing this. I compared many times original project and this fork but do not see any issue with the loss calculation... Thanks for this challenge, I try to help you as well. Please give me some time to figured it out.

**Comment by yaoanderson** [comment]
Thank you so much sowson, I expect your feedback any time.
I tried:
learning rate 0.001, 0.0001
batch size and sups: 64/8, 64/16, 32/8
burnin: 0
random:1, 0
anchors: voc default, from my kmean 
....
But any combination works bad based on your framework, all the avg loss just can down to about 10, so this is my case, help me please

**Comment by yaoanderson** [comment]
hi @sowson, is there any update for your research?

**Comment by sowson** [comment]
Here you go...
[yolov2-n.cfg.txt](https://github.com/sowson/darknet/files/3245136/yolov2-n.cfg.txt)
Thanks!

**Comment by yaoanderson** [comment]
thanks so much sowson.
How about your training progress and some change checkpoints as I mentioned above ?

**Comment by yaoanderson** [comment]
@sowson I have seen your yolov2.txt, but I found maybe [region]
anchors = 6,14, 70,82, 176,190, 291,375, 382,377 is not expected, and I have searched articles about anchors in yolov2 which suggest to value from 0.0 to 13.0, but yours is not, so can you train your data to get good result ? I just double confirm with you about my confuse. Thank you.

**Comment by sowson** [comment]
This is default yolov3-voc.cfg from the original author and this repository as well. I only changed
from:
learning_rate=0.001
burn_in=1000
to:
learning_rate=0.0001
burn_in=100
Of course, on top of the file, training part and testing part is switched to train.
Thanks, and enjoy!

**Comment by yaoanderson** [comment]
Fix, Thanks so much sowson !
